src/lib/arch/aarch64/ExceptionHandler.cc

```diff
@@ -713,7 +712,8 @@ bool ExceptionHandler::init() {
          static_cast<uint16_t>(arch.getSystemRegisterTag(ARM64_SYSREG_SVCR))});
     regValues.push_back(RegisterValue(newSVCR, 8));
 
-    ProcessStateChange stateChange = {ChangeType::REPLACEMENT, regs, regValues};
+    ProcessStateChange stateChange = {
+        ChangeType::REPLACEMENT, regs, regValues, {}, {}};
```

Comment:
This may be layover from testing. For full correctness all ProcessStateChanges should probably be initialised in this way. Have removed and will check the CI